internal/legacy/transform/transform.go

```diff
@@ -474,6 +476,16 @@ func ConvertSnykSchemaFindingsToLegacy(params *SnykSchemaToLegacyParams) (*defin
 	}
 	res.Remediation = remSummary
 
+	policy, err := cmdutil.GetLocalPolicy(ctx)
+	if err != nil {
+		return nil, params.ErrFactory.NewLegacyJSONTransformerError(fmt.Errorf("failed to get local policy: %w", err))
+	}
+	policyStr, err := ExtendLocalPolicyFromSchema(policy, params.Findings)
```

Comment:
(optional) extendedPolicyStr as a variable name and ExtendLocalPolicyFromFindings as a function name might be better